pom.xml

```diff
@@ -75,9 +75,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.bukkit</groupId>
-            <artifactId>bukkit</artifactId>
-            <version>1.15.1-R0.1-SNAPSHOT</version>
+            <groupId>org.spigotmc</groupId>
```

Comment:
Why are you switching from bukkit to spigot here? Is there a dependency on spigot?

Reply:
Mainly because I wanted to compile against 1.19 to make sure there's no deprecated stuff being called, and 1.15.1 is the last version of bukkit that seems to exist - I think I remember the spigot makers said they'd drop bukkit in 1.16. Also, I'm not sure if sqlite is in bukkit. Which might be irrelevant but then it might not, as the code doesn't directly reference sqlite3, it just pulls it in via jdbc.

So that change was more of a "bring it in line with everything else" thing than an actual necessity.

Reply:
Works for me! Thanks.